src/GafferSceneUIModule/LightEditorBinding.cpp

```diff
@@ -198,9 +198,20 @@ class InspectorColumn : public PathColumn
 			/// return a colour for `Role::Value`?
 			result.icon = runTimeCast<const Color3fData>( inspectorResult->value() );
 			result.background = g_sourceTypeColors.at( (int)inspectorResult->sourceType() );
+			std::string toolTip;
 			if( auto source = inspectorResult->source() )
 			{
-				result.toolTip = new StringData( "Source : " + source->relativeName( source->ancestor<ScriptNode>() ) );
+				toolTip = "Source : " + source->relativeName( source->ancestor<ScriptNode>() );
+			}
+
+			if( runTimeCast<const IECore::BoolData>( result.value ) )
+			{
+				toolTip += !toolTip.empty() ? "\nDouble-click to toggle" : "Double-click to toggle";
```

Comment:
I think the tooltip would be a little more readable if we used `\n\n` to include a blank line between the "Source" part and the "Things you can do" part. That would match similar tooltips in the HierarchyView and NodeEditor.

Reply:
Fixed in 1b23e6defb06f657e1ea0fa85179ad9578cf3cc8